OpenRA.Game/Sound/Sound.cs

```diff
@@ -113,6 +113,16 @@ public void StopAudio()
 			soundEngine.StopAllSounds();
 		}
 
+		public void MuteAudio()
+		{
+			soundEngine.Volume = 0f;
+		}
+
+		public void UnmuteAudio()
+		{
+			soundEngine.Volume = 1f;
```

Comment:
@Phrohdoh Unless I am mistaken I don't think the player can set overall volume? The player can adjust the sound, music and video volume independently. This PR respects those settings.

Please correct me if I'm wrong.